test/e2e/set/scyllacluster/scyllacluster_shardawareness.go

```diff
@@ -59,8 +66,6 @@ var _ = g.Describe("ScyllaCluster", func() {
 
 		hosts := getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
 		o.Expect(hosts).To(o.HaveLen(1))
-		di := insertAndVerifyCQLData(ctx, hosts)
```

Comment:
why?

Reply:
it creates additional connection and writes data to cluster which makes debugging harder. CQL session required by test is created below.